README.md

```diff
@@ -70,19 +72,48 @@ The Kata Containers project is governed according to the ["four opens"](https://
+For contributors, there are several roles relevant to project governance:
 
 ### Contributor
 
 A Contributor to the Kata Containers project is someone who has had code merged within the last 12 months. Contributors are eligible to vote in the Architecture Committee elections. Contributors have read only access to the Kata Containers repos on GitHub.
 
-### Maintainer
+### Committer
 
-A Maintainer has the ability to merge code into the Kata Containers project. Maintainers are active Contributors and participants in the projects. In order to become a Maintainer, you must be nominated and approved by the established Maintainers. Maintainers have write access to the Kata Containers repos on GitHub.
+Kata Containers Committers (as defined by the [kata-containers-committer team](https://github.com/orgs/kata-containers/teams/kata-containers-committer))
+have the ability to merge code into the Kata Containers project.
+Committers are active Contributors and participants in the projects. In order to become a Committer, you must be nominated by established Committer and approved by quorum of the active Architecture committee via an issue against the community repo
+e.g. https://github.com/kata-containers/community/issues/403. Committers have write access to the Kata Containers repos on GitHub, which
+gives the ability to approve PRs, trigger the CI and merge PRs.
+
+One of the requirements to be a committer is that you are an active contributor to the project as adjudged by the above criteria. Assessing the list of active contributors happens twice a year,
+lining up with the Architecture Committee election cycle. At that time, people who are in the kata-containers-committer team will also be reviewed, and a list of people, who are on the team,
+but who haven't been an active contributor in the last 12 months will be created and shared with the Architecture Committee and community.
+After a short review period, to allow time to check for any errors, inactive team members will be removed.
+
+> [!Note]
+> See [issue #413](https://github.com/kata-containers/community/issues/413) for a potential change in how active contribution is assessed.
+
+### Admin
+
+Kata Containers Admins (as defined by the [kata-containers-admin team](https://github.com/orgs/kata-containers/teams/kata-containers-admin) have admin access to
+the kata-containers repo, allowing them to do actions like, change the branch protection rules for repositories, delete a repository and manage the access of others.
+The Admin group is intentionally kept small, however, individuals can be granted temporary admin access to carry out tasks, like creating a secret that is used in a particular CI infrastructure.
+The Admin list is reviewed and updated after each Architecture Committee election and typically contains:
+- The Architecture Committee
+- Optionally, some specific people that the Architecture Committee agree on adding for a specific purpose (e.g. to manage the CI)
```

Comment:
I understand we're taking this a security measure to prevent something to happen.  However, the community has always been based on trust and work being done.

Having this feels like we're shooting ourselves in the foot, as either you're an AC member, member of OIF, or you depend on their own criteria (regardless of them being actually active or not) to decide who'll be actually helping the project.

Now, if we want to go through a formal process, I'd **strongly** recommend that any change in the admin side would to go through a vote and the majority of the admins would have to agree before any change happens (including secrets being added, machines being added, PRs being merged, etc).  The malicious actor may come from anywhere, including AC / OIF.